portal-web/test/functional/com/liferay/portalweb/macros/ManagementBar.macro

```diff
@@ -45,7 +45,7 @@ definition {
 
 		WaitForLiferayEvent.initializeLiferayEventLog();
 
-		MenuItem.click(menuItem = ${menuItem});
+		Click.pauseClickAt(locator1 = "ManagementBar#FILTER_AND_ORDER");
 	}
```

Comment:
When clicking to sort and selecting the desired option, the macro `MenuItem.click(menuItem = ${menuItem});` continues to click on the desired option to sort and the modal is not closed.
Clicking the filter button after selecting the chosen option closes the modal and is faster and more specific.